src/main/java/Duke.java

```diff
@@ -1,35 +1,36 @@
 import java.util.Scanner;
-import java.util.ArrayList;
+//import java.util.ArrayList;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello! I'm Duke\n" + "What can I do for you?");
-        int x=0,y,l;
-        ArrayList<String> list=new ArrayList<>();
+        int x=0,y,l,d;
```

Comment:
Not very readable and not descriptive variable names. Improve by coming up with appropriate naming.

Reply:
On first glance, we do not know what is int x,y,l,d. Maybe it would be best to add in a comment do describe what each of them is used for.
A more descriptive naming will be the best!